infra/k8s/demo/web.yml

```diff
@@ -25,8 +25,7 @@ metadata:
   labels:
     k8s-app: demo
 spec:
-  # single replica because of volume mounts
-  replicas: 1
+  replicas: 3
```

Comment:
Puur voor mijn begrip: waarom wordt replicas nu op 3 gezet terwijl het eerst op 1 stond?

Reply:
Haven heeft nog geen support voor `ReadWriteMany` volumes, wat betekent dat er slechts 1 pod een volume kan mounten. Door het verwijderen van het volume hebben we nu wel de mogelijkheid om meerdere pods te draaien (= meer replicas). Dit verhoogt de availability bij updates, door dat er altijd 1 pod in de lucht blijft terwijl de rest geüpdatet wordt.